src/microbe_stage/ColonyCompoundBag.cs

```diff
@@ -12,7 +12,7 @@ public ColonyCompoundBag(MicrobeColony colony)
     }
 
     [JsonProperty]
-    private MicrobeColony Colony { get; set; }
+    private MicrobeColony Colony { get; }
```

Comment:
I thought saving won't work with properties with no setters?

Reply:
It actually works (and now that I've improved the system, I think we should move towards this style) if there is a constructor (either marked with the json constructor property or the first one) that takes a parameter of that name.